app/components/ucsf/ClinicianActions/ClinicianActions.module.scss

```diff
@@ -0,0 +1,50 @@
+@import '~styles/utils/_helpers.scss';
+
+.clinicianActions {
+  position: relative;
+  width: 850px;
+}
+
+.title {
+  font-size: 26px;
+  font-weight: 700;
+  line-height: 1;
+  color: #000;
+}
+
+.actionLists {
+  display: grid;
+  gap: 16px;
+  padding: 32px 0 40px;
+}
+
+.actionListContainer {
+  padding: 16px;
+  font-size: 20px;
+  box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.2);
+  border-radius: 4px;
+}
+
+.actionType {
+  font-weight: 700;
+  line-height: 1;
+  padding-bottom: 1em;
+}
+
+.actionList {
+  list-style: initial;
+  list-style-position: inside;
+  padding-left: 10px;
+}
+
+.buttonBar {
+  display: flex;
+  justify-content: end;
+}
+
+.closeBtn {
+  width: 128px;
+  height: 53px;
+  border-radius: 3px;
+  font-size: 16px;
```

Comment:
Do we actually need to set all these properties? From a design perspective, it would be better if the buttons on our site were as consistent as possible and that we stick to as few unique styles as possible.

Regarding width specifically, it looks like our default button styles already choose 120px as the default width. I'm not sure it's worth overriding the width when we have an existing width that is so close. For height, we have a similar thing where the existing button styles set 56px as the default, so I don't know if overriding it to 53px is necessary.

In addition, for height, we are kind of already redundantly setting it in the base button component, since the height is redundant with (top padding + bottom padding + line height * font-size), and setting both sets of properties makes it less clear what the desired final height should be. That said, this particular issue affects the base button component and should probably be resolved in a separate PR. Just created https://github.com/ShelterTechSF/askdarcel-web/issues/1165 to track this.

Reply:
I've gone ahead and removed the width/height as well as `border-radius` and `font-size`. I have made a note to bring up button standardization during the next design meeting or two